joblib/test/test_numpy_pickle.py

```diff
@@ -231,6 +233,46 @@ def test_z_file():
         data_read = numpy_pickle.read_zfile(f)
     nose.tools.assert_equal(data, data_read)
 
+
+def test_compressed_pickle_python_2_3_compatibility():
```

Comment:
This test requires numpy, so you should do as the others in the same file that require numpy, and define it only conditionally (numpy is an optional dependency of joblib).